### ISSUE TITLE
networking: tricky to navigate to Firewall page

### ISSUE THREAD

**Issue by andreasn** [cockpit-project/cockpit#14303]
Cockpit version: 220
OS: RHEL
Page: Networking

I a usability study we ran in June with 5 participants, one participant had great difficulty finding the firewall page, and another spent quite some time hunting for it before finding it.
This could be because the panel in the Firewall section doesn't look clickable enough.
![Screenshot from 2020-06-30 17-56-53](https://user-images.githubusercontent.com/209895/86353178-25cbfb80-bc67-11ea-8f3b-c2baecb0e3e1.png)

**Comment by andreasn** [comment]
Note that all participants first instinct was to find firewall ports in the Networking page, it was only once they ended up there that they had trouble getting to the subpage.

**Comment by marusak** [comment]
Yeah, they often just looked at those 'Add Bond`, `Add Team`... buttons.
I feel like we should have similar button on the firewall panel saying something like `Manage firewall`.

**Comment by garrett** [comment]
I'm working on a redesign of the page which incorporates a fix for the discoverability of the firewall section. I discussed it with @andreasn in our meeting today and I'll try to get it closer to an implementable state next week (as it's the weekend for me now).

Here's a preview:

![network-overview](https://user-images.githubusercontent.com/10246/86387721-dea82f80-bc93-11ea-9625-2c2307aec77c.png)

(Perhaps this is even enough to start?)

Despite the graphs being absent on the mockup, we're still going to have them on the page. I just haven't incorporated them yet.

**Comment by andreasn** [comment]
How should we lay it out on the page?

Version one - no cards:
![network-page-ver1](https://user-images.githubusercontent.com/209895/87562712-9985f380-c6be-11ea-9efb-52c92e3ff14b.png)

Version two - two different cards, one for graphs, one for the rest
![network-page-ver2](https://user-images.githubusercontent.com/209895/87562797-b8848580-c6be-11ea-8b68-20b22a515955.png)

Version three - split it all up into cards
![network-page-ver3](https://user-images.githubusercontent.com/209895/87562825-c2a68400-c6be-11ea-9ac4-a7c75a787efa.png)

I'm good with either approach, so let me know what one you like best.

**Comment by garrett** [comment]
Oh, I hadn't considered using cards here. They're generally used for splitting things up horizontally as well, but they do reinforce which widgets belong to which section, as seen in version 3. Option 2 is odd — having a everything in one card doesn't make much sense, IMO.

(I still think the sending / receiving graphs are way too large. They should be de-emphasized... in another PR.)

TL;DR: Let's go with 3?

**Comment by andreasn** [comment]
Sounds good!

**Comment by marusak** [comment]
Agree, option 3 seems like the best one to me as well.

**Comment by martinpitt** [comment]
Thanks for the design! I put that into our quarterly plan as COCKPIT-677, this appears relatively easy to to implement.

**Comment by andreasn** [comment]
The firewall issue is fixed in #14509. @garrett's design had a MAC address entry, but I think that's outside the scope of this issue. So closing.

**Comment by garrett** [comment]
Wait. Was the issue fixed? I don't see a button to edit the firewall on the redesigned page. (Which was the entire point.)

**Comment by KKoukiou** [comment]
The button is there.
![Screen Shot 2020-09-01 at 10 41 36](https://user-images.githubusercontent.com/14921356/91827886-bc08a780-ec3f-11ea-9165-3506ada929f8.png)

**Comment by garrett** [comment]
Oh, whoops. Wrong branch. :facepalm: Sorry.

**Comment by garrett** [comment]
I had some sort of weird hybrid:

![Screenshot_2020-09-01 Networking - garrett Rain](https://user-images.githubusercontent.com/10246/91828061-fa05cb80-ec3f-11ea-8115-49e15f5439b6.png)
